README.md

```diff
@@ -68,3 +68,41 @@ Use the [https-proxy](https://github.com/nulib/aws-developer-environment#conveni
 ```
 https-proxy start 3002 3000
 ```
+
+## Writing Documentation
+
+API documentation is automatically regenerated and deployed on pushes to the staging and production branches. The documentation is in two parts:
+
+### Regular Docs
+
+The `docs` directory contains a standard `mkdocs` project, which can be edited using the same tools and format as the main [Repository Documentation](http://docs.rdc.library.northwestern.edu/#contributing).
+
+In a nutshell:
+
+1. Clone this project into a working directory (which you probably already have).
+2. Edit the Markdown files in the `docs/docs` directory.
+3. To run `mkdocs` locally and preview your work:
+   ```shell
+   pip install poetry # only has to be done once
+   cd docs
+   poetry install
+   sg open all 8000
+   poetry run mkdocs serve -a 0.0.0.0:8000
+   ```
+   Docs will be accessible at https://[DEV_PREFIX].dev.rdc.library.northwestern.edu:8000/
```

Comment:
I think this should this be `http` instead of `https`, unless you want to add a step to use `https-proxy`?

Reply:
Good catch. Copypasta error.